exchanges/deribit/deribit_test.go

```diff
@@ -0,0 +1,2397 @@
+}
+
+func TestUnmarshalCancelResponse(t *testing.T) {
+	t.Parallel()
+	data := `{ "currency": "BTC", "type": "trigger", "instrument_name": "ETH-PERPETUAL", "result": [{ "web": true, "triggered": false, "trigger_price": 1628.7, "trigger": "last_price", "time_in_force": "good_til_cancelled", "stop_price": 1628.7, "replaced": false, "reduce_only": false, "price": "market_price", "post_only": false, "order_type": "stop_market", "order_state": "untriggered", "order_id": "ETH-SLTS-250756", "max_show": 100, "last_update_timestamp": 1634206091071, "label": "", "is_rebalance": false, "is_liquidation": false, "instrument_name": "ETH-PERPETUAL", "direction": "sell", "creation_timestamp": 1634206000230, "api": false, "amount": 100 }] }`
+	var resp OrderCancelationResponse
+	err := json.Unmarshal([]byte(data), &resp)
+	assert.NoError(t, err)
+}
+
+func TestGetLockedStatus(t *testing.T) {
+	t.Parallel()
+	_, err := d.GetLockedStatus(context.Background())
+	assert.NoError(t, err)
+}
+
+func TestSayHello(t *testing.T) {
+	t.Parallel()
+	_, err := d.SayHello("Sami", "")
+	assert.NoError(t, err)
+}
+
+func TestGetCancelOnDisconnect(t *testing.T) {
+	t.Parallel()
+	sharedtestvalues.SkipTestIfCredentialsUnset(t, d)
+	_, err := d.GetCancelOnDisconnect(context.Background(), "account")
+	assert.NoError(t, err)
+	_, err = d.WsRetrieveCancelOnDisconnect("connection")
+	assert.NoError(t, err)
+}
+
+func TestDisableCancelOnDisconnect(t *testing.T) {
+	t.Parallel()
+	sharedtestvalues.SkipTestIfCredentialsUnset(t, d, canManipulateRealOrders)
+	_, err := d.DisableCancelOnDisconnect(context.Background(), "connection")
+	assert.NoError(t, err)
+	_, err = d.WsDisableCancelOnDisconnect("connection")
+	assert.NoError(t, err)
+}
+
+func TestEnableCancelOnDisconnect(t *testing.T) {
+	t.Parallel()
+	sharedtestvalues.SkipTestIfCredentialsUnset(t, d, canManipulateRealOrders)
+	_, err := d.EnableCancelOnDisconnect(context.Background(), "account")
+	assert.NoError(t, err)
+	_, err = d.WsEnableCancelOnDisconnect("connection")
+	assert.NoError(t, err)
+}
+
+func TestLogout(t *testing.T) {
+	t.Parallel()
+	sharedtestvalues.SkipTestIfCredentialsUnset(t, d, canManipulateRealOrders)
+	err := d.WsLogout(true)
+	assert.NoError(t, err)
+}
+
+func TestExchangeToken(t *testing.T) {
+	t.Parallel()
+	sharedtestvalues.SkipTestIfCredentialsUnset(t, d)
+	_, err := d.ExchangeToken(context.Background(), "1568800656974.1CWcuzUS.MGy49NK4hpTwvR1OYWfpqMEkH4T4oDg4tNIcrM7KdeyxXRcSFqiGzA_D4Cn7mqWocHmlS89FFmUYcmaN2H7lNKKTnhRg5EtrzsFCCiuyN0Wv9y-LbGLV3-Ojv_kbD50FoScQ8BDXS5b_w6Ir1MqEdQ3qFZ3MLcvlPiIgG2BqyJX3ybYnVpIlrVrrdYD1-lkjLcjxOBNJvvUKNUAzkQ",
+		1234)
+	assert.NoError(t, err)
+	_, err = d.WsExchangeToken("1568800656974.1CWcuzUS.MGy49NK4hpTwvR1OYWfpqMEkH4T4oDg4tNIcrM7KdeyxXRcSFqiGzA_D4Cn7mqWocHmlS89FFmUYcmaN2H7lNKKTnhRg5EtrzsFCCiuyN0Wv9y-LbGLV3-Ojv_kbD50FoScQ8BDXS5b_w6Ir1MqEdQ3qFZ3MLcvlPiIgG2BqyJX3ybYnVpIlrVrrdYD1-lkjLcjxOBNJvvUKNUAzkQ",
+		1234)
+	assert.NoError(t, err)
+}
+
+func TestForkToken(t *testing.T) {
```

Comment:
Error:      	Received unexpected error:
        	            	session_name is required
        	Test:       	TestForkToken

Your func has the following:
```go
if sessionName != "" {
		return nil, errors.New("session_name is required")
	}
```
😄 please invert